SlotItem/SlotItem.module.less

```diff
@@ -11,6 +11,14 @@
 };
 
 .slotItem {
+	&.flex {
+		display: flex;
```

Comment:
let's use disply:flexall the time for .slotItem and remove the other added styling